user/deployment/custom.md

```diff
@@ -9,22 +9,6 @@ You can easily deploy to your own server the way you would deploy from your loca
 You may choose the [Script provider](/user/deployment/script/) instead, as it provides
 easier flexibility with conditional deployment.
 
-### FTP
```

Comment:
Perhaps we should direct users to the new norm: "Do not use FTP".

Reply:
@BanzaiMan I'm going to back out that deletion and provide an SFTP example instead, I think :+1:

Reply:
That sounds better!